utilities/email_helper.py

```
@@ -0,0 +1,322 @@
+                    normalized_emails.append(normalized)
+        
+        # Update league with normalized, deduplicated emails
+        await db.execute(
+            update(League)
+            .where(League.id == league_id)
+            .values(league_member_emails=json.dumps(normalized_emails))
+        )
+        await db.commit()
+        
+        print(f"Set {len(normalized_emails)} normalized, deduplicated emails for league {league_id}")
+        return True
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)